website/sidebars.js

```diff
@@ -45,6 +45,21 @@ module.exports = {
                 'topics/managing-constraints',
                 'topics/proxy-hosting',
                 'topics/data-collection',
+                {
+                    type: 'category',
+                    label: 'Feature Flags Migrations',
```

Comment:
Should this be plural or singular? Plural sounds a little strange to me, but I'll leave it up to you.

```suggestion
                    label: 'Feature Flag Migrations',
```